File: from-0001-to-0100/0062-unique-paths/solution.go
func uniquePaths(m int, n int) int {
    dp := make([][]int, m)
    for i := range dp {
        dp[i] = make([]int, n)
    }

    for r := 0; r < m; r++ {
        for c := 0; c < n; c++ {
            if r == 0 || c == 0 {
                dp[r][c] = 1
            } else {
                dp[r][c] = dp[r-1][c] + dp[r][c-1]
            }
        }
    }
    return dp[m-1][n-1]

File: from-0001-to-0100/0062-unique-paths/solution.py
class Solution:
    def uniquePaths(self, m: int, n: int) -> int:
        dp = [ [1 if _c == 0 or _r == 0 else 0 for _c in range(n)] for _r in range(m) ]
        
        for r in range(1, m):
            for c in range(1, n):
                dp[r][c] = dp[r-1][c] + dp[r][c-1]
                
        return dp[m-1][n-1]